src/components/settings/SettingsLMSTab/tests/BlackboardConfig.test.jsx

```diff
@@ -3,18 +3,44 @@ import {
   render, fireEvent, screen,
 } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
+import { act } from 'react-dom/test-utils';
 
 import BlackboardConfig from '../LMSConfigs/BlackboardConfig';
 import { INVALID_LINK, INVALID_NAME } from '../../data/constants';
 import LmsApiService from '../../../../data/services/LmsApiService';
 
-jest.mock('../../../../data/services/LmsApiService');
+jest.mock('../../data/constants', () => ({
+  ...jest.requireActual('../../data/constants'),
+  LMS_CONFIG_OAUTH_POLLING_INTERVAL: 1,
+}));
+window.open = jest.fn();
```

Comment:
would to good to spy on this or sth so you can verify we do window.open when expected with correct url